lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -304,14 +310,32 @@ def get_credentials()
         # authenticate API requests.
         def get_updater_proc()
           return Signet::OAuth2::Client.new(
-            token_credential_uri: 'https://www.googleapis.com/oauth2/v3/token',
+            token_credential_uri: "https://www.googleapis.com/oauth2/v3/token",
             client_id: @config.client_id,
             client_secret: @config.client_secret,
             refresh_token: @config.refresh_token,
-            scope: ['https://www.googleapis.com/auth/adwords']
+            scope: ["https://www.googleapis.com/auth/adwords"]
           ).updater_proc
         end
 
+        # Provides a Google::Auth::Credentials initialized with a keyfile
+        # specified in the config.
+        def get_service_account_credentials()
+          raise 'config.impersonate required if keyfile specified' unless @config.impersonate
+          keyfile = File.read(@config.keyfile)
+          keyfile = JSON.parse(keyfile, symbolize_names: true)
```

Comment:
What do you think about not using symboilze_names here, instead just using string keys?